src/styles/index.scss

```diff
@@ -1,3 +1,60 @@
+@import './utils/variable';
```

Comment:
Ensure that the file './utils/variable' exists and is correctly imported. Also, verify that the variables used in this file are defined in the imported file.